src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -454,6 +472,28 @@ export function registerTerminalActions() {
 		}
 	});
 
+	registerActiveInstanceAction({
+		id: TerminalCommandId.CopyLastCommandAndLastCommandOutput,
+		title: { value: localize('workbench.action.terminal.copyLastCommandAndLastCommandOutput', 'Copy Last Command And Last Command Output'), original: 'Copy Last Command And Last Command Output' },
+		precondition: ContextKeyExpr.or(TerminalContextKeys.processSupported, TerminalContextKeys.terminalHasBeenCreated),
+		run: async (instance, c, accessor) => {
+			const clipboardService = accessor.get(IClipboardService);
+			const commands = instance.capabilities.get(TerminalCapability.CommandDetection)?.commands;
+			if (!commands || commands.length === 0) {
+				return;
+			}
+			const command = commands[commands.length - 1];
+			if (!command?.hasOutput()) {
+				return;
+			}
+			const output = command.getOutput();
+			if (isString(output)) {
+				await clipboardService.writeText(`${command.command !== '' ? command.command + '\n' : ''}${output}`);
```

Comment:
I copied this logic from https://github.com/microsoft/vscode/pull/192217/files#diff-f26cf22b79d035fac45646c72f194cc92cf9629bd5cb58e8d8a39d220bb0d5dcR389.

Reply:
Maybe it's worth extracting this duplicated logic to a utility somewhere?